src/test/java/underdogs/devbie/question/controller/QuestionControllerTest.java

```diff
@@ -0,0 +1,156 @@
+package underdogs.devbie.question.controller;
+
+import static org.hamcrest.core.StringContains.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static underdogs.devbie.auth.controller.AuthControllerTest.*;
+import static underdogs.devbie.question.acceptance.QuestionAcceptanceTest.*;
+import static underdogs.devbie.user.domain.UserTest.*;
+
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import underdogs.devbie.MvcTest;
+import underdogs.devbie.auth.controller.interceptor.BearerAuthInterceptor;
+import underdogs.devbie.auth.controller.resolver.LoginUserArgumentResolver;
+import underdogs.devbie.question.dto.QuestionCreateRequest;
+import underdogs.devbie.question.dto.QuestionResponse;
+import underdogs.devbie.question.dto.QuestionResponses;
+import underdogs.devbie.question.dto.QuestionUpdateRequest;
+import underdogs.devbie.question.service.QuestionService;
+import underdogs.devbie.user.domain.User;
+
+@WebMvcTest(QuestionController.class)
+class QuestionControllerTest extends MvcTest {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @MockBean
+    private QuestionService questionService;
+
+    @MockBean
+    private BearerAuthInterceptor bearerAuthInterceptor;
+
+    @MockBean
+    private LoginUserArgumentResolver loginUserArgumentResolver;
```

Comment:
```suggestion
    @Autowired
    private BearerAuthInterceptor bearerAuthInterceptor;

    @Autowired
    private LoginUserArgumentResolver loginUserArgumentResolver;
```
로그인 했다고 가정되는데요! 찐 Bean을 써야하지 않나 싶어요.

Reply:
setUp부분에 보면 Interceptor와 resolver에서 mock으로 처리해야할 부분이 존재합니다.
그렇지 않으면, 여기서도 인수테스트 처럼 진짜 토큰을 가지고 테스트해야 합니다.